modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -38,7 +49,33 @@ import AposCellMixin from 'Modules/@apostrophecms/ui/mixins/AposCellMixin';
 
 export default {
   name: 'AposCellLabels',
-  mixins: [ AposCellMixin ]
+  mixins: [ AposCellMixin ],
+  data() {
+    return {
+      customCellIndicators: apos.schema.customCellIndicators
+    };
+  },
+  methods: {
+    evaluateConditions(conditions = {}) {
+      try {
+        return Object.entries(conditions).every(([ property, expected ]) => {
+          const properties = property.split('.');
+
+          const draftProp = properties.reduce((acc, cur) => {
+            if (Object.hasOwn(acc, cur)) {
+              return acc[cur];
+            }
+
+            throw new Error(`Property not found in draft document: ${property}`);
```

Comment:
throwing here to be sure we won't have false positive.

Reply:
I think you can import lodash/get and use it `_.get(this.draft, property)`. If the draft prop doesn't exist, lodash will just return undefined, which is fine. 
No problem with loading lodash/get because fields already use it (array retrieve the tab label via lodash.get).

Reply:
I prefer to avoid lodash when it's not complex stuff most of the time.
What bother you is the use of try catch and error to get out of the loop?